tests/unittest_manager.py

```diff
@@ -362,6 +362,19 @@ def test_brain_plugins_reloaded_after_clearing_cache(self) -> None:
         inferred = next(format_call.infer())
         self.assertIsInstance(inferred, Const)
 
+    def test_builtins_inference_after_clearing_cache(self) -> None:
+        astroid.MANAGER.clear_cache()
+        isinstance_call = astroid.extract_node("isinstance(1, int)")
+        inferred = next(isinstance_call.infer())
+        self.assertIs(inferred.value, True)
+
+    def test_builtins_inference_after_clearing_cache_manually(self) -> None:
+        # Not recommended to manipulate this, so we detect it and call clear_cache() instead
```

Comment:
In trying to cover the missed lines I just decided to support this, since it's an easy mistake to make.